Content.Server/GameTicking/GameTicker.StatusShell.cs

```diff
@@ -44,9 +47,12 @@ private void GetStatusResponse(JsonNode jObject)
                 jObject["name"] = _baseServer.ServerName;
                 jObject["map"] = _gameMapManager.GetSelectedMap()?.MapName;
                 jObject["round_id"] = _gameTicker.RoundId;
-                jObject["players"] = _cfg.GetCVar(CCVars.AdminsCountInReportedPlayerCount)
-                    ? _playerManager.PlayerCount
-                    : _playerManager.PlayerCount - _adminManager.ActiveAdmins.Count();
+                // Harmony Queue Start
+                // jObject["players"] = _cfg.GetCVar(CCVars.AdminsCountInReportedPlayerCount)
+                //     ? _playerManager.PlayerCount
+                //     : _playerManager.PlayerCount - _adminManager.ActiveAdmins.Count();
+                jObject["players"] = _joinQueue.ActualPlayersCount;
```

Comment:
Please keep the functionality to not count admins in the launcher player count
```suggestion
                jObject["players"] = _cfg.GetCVar(CCVars.AdminsCountInReportedPlayerCount)
                    ? _joinQueue.ActualPlayersCount
                    : _joinQueue.ActualPlayersCount - _adminManager.ActiveAdmins.Count();
```